e2e/helper.go

```diff
@@ -0,0 +1,123 @@
+package e2e
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+	"text/template"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+const (
+	WorkloadIdentityNamespace = "azure-workload-identity-system"
+)
+
+// Env variables required for the setup and cleanup
+var (
+	AzureADTenantID               = os.Getenv("AZURE_SP_TENANT")
+	AzureRunWorkloadIdentityTests = os.Getenv("AZURE_RUN_WORKLOAD_IDENTITY_TESTS")
+)
+
+var (
+	Kc *kubernetes.Clientset
+)
+
+func ParseCommand(cmdWithArgs string) *exec.Cmd {
+	split := strings.Fields(cmdWithArgs)
+
+	return exec.Command(split[0], split[1:]...)
+}
+
+func GetKubernetesClient(t *testing.T) *kubernetes.Clientset {
+	if Kc != nil {
+		return Kc
+	}
+
+	kubeConfig, err := config.GetConfig()
+	require.NoErrorf(t, err, "ensure that we are logged onto a kubernetes cluster - %s", err)
+
+	Kc, err = kubernetes.NewForConfig(kubeConfig)
+	require.NoErrorf(t, err, "ensure that we are logged onto a kubernetes cluster - %s", err)
+
+	return Kc
+}
+
+func CreateNamespace(t *testing.T, kc *kubernetes.Clientset, nsName string) {
+	namespace := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: nsName,
```

Comment:
We should apply the label here, see:
https://github.com/kedacore/keda/blob/af28b7b5cf79a408198dfcf4c798c2bbddb8b5f3/tests/scalers/helpers.ts#L63-L64

Reply:
Done.